[docs] Fixes and improvements to Building for TV (#27078)
expo/expo: docs/pages/guides/building-for-tv.mdx

> **Info** If you are starting with an SDK 49 project, you will need to [upgrade to SDK 50 or higher](/workflow/upgrading-expo-sdk-walkthrough/) before proceeding through the following steps.

## Supported modules

At this time, TV applications work with the following packages and APIs listed in our Expo API docs.

- [AppleAuthentication](/versions/latest/sdk/apple-authentication/)
- [Application](/versions/latest/sdk/application/)
- [Asset](/versions/latest/sdk/asset/)
- [AV](/versions/latest/sdk/av/)
- [BlurView](/versions/latest/sdk/blur-view/)
- [Crypto](/versions/latest/sdk/crypto/)
- [Device](/versions/latest/sdk/device/)
- [FileSystem](/versions/latest/sdk/filesystem/)
- [FlashList](/versions/latest/sdk/flash-list/)
- [Font](/versions/latest/sdk/font/)
- [Image](/versions/latest/sdk/image)
- [KeepAwake](/versions/latest/sdk/keep-awake/)
- [LinearGradient](/versions/latest/sdk/linear-gradient/)
- [Localization](/versions/latest/sdk/localization/)
- [Reanimated](/versions/latest/sdk/reanimated/)
- [SplashScreen](/versions/latest/sdk/splash-screen/)
- [Updates](/versions/latest/sdk/updates/)

> TV also works with react-navigation, react-native-video, and many other commonly used 3rd party React Native libraries.

Reviewer: ```suggestion
TV also works with `react-navigation`, `react-native-video,` and many other commonly used third-party React Native libraries.
```

- Let's highlight the library names and format how we say "third-party"

## Add the TV config plugin

<Terminal cmd={['npx expo install --dev @react-native-tvos/config-tv']} />

When installed, the plugin will modify the project for TV when either:

- The environment variable `EXPO_TV` is set to `1`
- The plugin parameter `isTV` is set to `true`

> Make sure that this plugin appears in `app.json`:

Reviewer: ```suggestion
Verify that this plugin appears in **app.json**:
```
- (Unrelated to this PR) bet let's make the file name bold as per our [writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)
- Also, let's improve the verbiage here

```json app.json
{
  "plugins": [
    "@react-native-tvos/config-tv"
  ]
}
```

>  To see additional information on the plugin's actions during prebuild, you can set [debug environment variables](https://github.com/debug-js/debug#conventions) before running prebuild. (See also our documentation on [Expo CLI environment variables](/more/expo-cli/#environment-variables).)

Reviewer: ```suggestion
 To see additional information on the plugin's actions during prebuild, you can set [debug environment variables](https://github.com/debug-js/debug#conventions) before running prebuild. (See also our documentation on [Expo CLI environment variables](/more/expo-cli/#environment-variables).)
```

Is there are specific reason we need to point to Expo CLI environment variables table here?

Author: That table is where we document `DEBUG=expo:*`... I couldn't find a way to link to that specific row in the table though.

> <Terminal cmd={[
>   'export DEBUG=expo:* # see all Expo CLI and config plugin debug information',
>   'export DEBUG=expo:react-native-tvos:config-tv # see only debug information for the TV plugin']}
> />

Reviewer: ```suggestion
<Terminal cmd={[
  '# See all Expo CLI and config plugin debug information'
  'export DEBUG=expo:*',
  '',
  '# See only debug information for the TV plugin'
  'export DEBUG=expo:react-native-tvos:config-tv']}
/>
```

Let's separate out the comment for each command here (we usually follow this pattern in the Terminal component).
